packages/rrweb/src/replay/index.ts

```diff
@@ -1219,6 +1219,14 @@ export class Replayer {
             if (isSync) {
               if (d.type === MouseInteractions.TouchStart) {
                 pointer.touchActive = true;
+
+                // prevents multiple touch circles from staying on the screen
+                // when the user seeks by breadcrumbs
+                Object.values(this.pointers).forEach((p) => {
+                  if (p !== pointer && !p.touchActive) {
+                    p.touchActive = false;
```

Comment:
Hmm since the condition is `!p.touchActive`, wouldn't this mean `touchActive` is already false?

Reply:
if `p.touchActive === true` (this can be true for various pointers at the same time in the case of multitouch) then we don't want to set `touchActive = false`. otherwise we'd remove the pointer erroneously for multitouch. but `touchActive` can also be `null` for the pointers that aren't active, so those are the cases we want to set `touchAcitve = false`

Reply:
Can we clarify this in a code comment? (and also what it means for it to be `null` vs `false`)